color_refiment_helper.py

```diff
@@ -12,8 +11,15 @@ def debug(*args):
         print(*args)
 
 
-def compare(s: "List", t: "List"):
-    return Counter(s) == Counter(t)
+def compare(s: "List", t: "List", my_key=None)-> bool:
+    """
+    Compares 2 lists and will do so on the sorted list
+    :param s: List
+    :param t: List
+    :param my_key: Key that you have to set to compare vertices: lambda object: object.property
+    :return: Boolean, True if the lists are the same
+    """
+    return sorted(s, key=my_key) == sorted(t, key=my_key)
```

Comment:
Ik denk dat hier (later) nog een optimalisatieslag kan worden gemaakt:
 - Sorteren kan _lazy_, dus het sorteren resulteert in een _generator_ i.p.v. een volledig gesorteerde lijst.
 - Deze methode kan elke `Iterable` accepteren, niet alleen lijsten.
 - Het vergelijken van de twee generators kan ook _lazy_: zodra er een element ongelijk is, dan is de hele vergelijking `False` en hoef je niet verder te vergelijken.

Reply:
Dan zou ik zeggen: maak maar vast een issue aan. Kan ook een lazy issue zijn waarbij we al deze dingen in een lijstje zetten en dan aanpakken als we denken dat dat iets oplevert :)

Reply:
#31